src/components/RandomBeer.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { useState, useEffect } from 'react'
+import Navbar from '../components/Navbar'
+import axios from 'axios';
+
+function RandomBeer() {
+    const [random, setRandom] = useState({});
+    const [oneBeerDet, setOneBeerDet] = useState({});
+
+    const getBeersApi = async () => {
+        try {
+            const response = await axios.get(`https://ih-beers-api2.herokuapp.com/beers/${random}`);
+            setRandom(response.data);
+        } catch (error) {
+            console.log(error);
+        }
+    };
+
+    useEffect(() => {
+        getBeersApi();
+    }, []);
+
+    return (
+        <div>
+            <Navbar />
+
+            <img src={oneBeerDet.image_url} alt="beer" />
```

Comment:
The `useState` that you created to store the response from the `API` is: `const [random, setRandom] = useState({})`
So here to access the information of the beer instead of `oneBeerDet.image_url` you have to do `random.image_url` because that's what you called it on the `useState`. Every time you used `oneBeerDet` on this page, you need to change it to `random`